DataFormats/Detectors/CTP/src/Configuration.cxx

```diff
@@ -857,8 +863,10 @@ CTPRunScalers CTPRunManager::getScalersFromCCDB(long timestamp, std::string run)
   auto ctpscalers = mgr.getSpecific<CTPRunScalers>(mCCDBPathCTPScalers, timestamp, metadata);
   if (ctpscalers == nullptr) {
     LOG(info) << "CTPRunScalers not in database, timestamp:" << timestamp;
+    ok = 0;
   } else {
     // ctpscalers->printStream(std::cout);
+    ok = 1;
   }
   return *ctpscalers;
```

Comment:
do you really need a copy of the object?
Note that for the DPL workflows we are outphasing custom access to CCDB objects in favour of DPL fetcher.

Reply:
This is utility function, it  is used only in macros, e.g.:
https://github.com/AliceO2Group/AliceO2/blob/e01f74f3624f955d44b4c6b189c448b9eb646b38/Detectors/CTP/macro/GetScalers.C#L45
never in workflow.